### PR TITLE
Ef tests electra

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

ef test updates

### CODE REVIEW

File: testing/ef_tests/src/handler.rs [hunk 1]
```diff
@@ -829,10 +833,7 @@ impl<E: EthSpec + TypeName> Handler for KzgInclusionMerkleProofValidityHandler<E
 
     fn is_enabled_for_fork(&self, fork_name: ForkName) -> bool {
         // Enabled in Deneb
-        fork_name != ForkName::Base
-            && fork_name != ForkName::Altair
-            && fork_name != ForkName::Bellatrix
-            && fork_name != ForkName::Capella
+        fork_name == ForkName::Deneb
```
Comment (pawanjay176) [inline testing/ef_tests/src/handler.rs:836]: shouldn't this be enabled for electra too?

**Review by pawanjay176 — APPROVED** [review]
Looks good. 
Are we thinking of updating to 1.5.0-alpha3 tests after merging this PR?

File: testing/ef_tests/src/handler.rs (hunk 1 above)
Comment (realbigsean) [reply testing/ef_tests/src/handler.rs:836]: there's no tests in this spec release for electra, but if they're added later it'll get cause by our accessed files check

**Comment by realbigsean** [comment]
>  Are we thinking of updating to 1.5.0-alpha3 tests after merging this PR? 

believe this is the `devnet-0` target so we should update to that release in mark's PR

**Comment by realbigsean** [comment]
@mergify queue

**Comment by realbigsean** [comment]
@mergify requeue

**Comment by realbigsean** [comment]
@mergify dequeue

**Comment by realbigsean** [comment]
@mergify requeue

**Comment by realbigsean** [comment]
@mergify queue

**Comment by realbigsean** [comment]
@mergify refresh

**Comment by realbigsean** [comment]
@mergify requeue

**Comment by realbigsean** [comment]
@mergify queue

**Pull request merged**